src/components/workspace/app-namespace.tsx

```diff
@@ -114,6 +144,10 @@ export default function AppNamespace(props: {
           <Typography color="primary" fontFamily='"Roboto Mono", ui-monospace, monospace'>{nameStr()}</Typography>
         }>
           <Typography color="secondary">Please input the trust anchor exported by cert-dump</Typography>
+          <IconButton onClick={() => setPopupOpen(!isPopupOpen())}>
+            <QRIcon color="primary"/>
+          </IconButton>
+          {isPopupOpen() && (<video ref={vid}></video>)}
```

Comment:
Please use [Show](https://www.solidjs.com/tutorial/flow_show) or [Switch](https://www.solidjs.com/tutorial/flow_switch). I think that is more readable than `&&`.